docker_digitaltwin/python_postgis/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8.8
```

Comment:
Good idea. But I've read somewhere that the -slim and -alpine images do not include a C compiler which means that some packages won't be compatible with these versions. If that's the case, we might have to keep that in mind while choosing an image.

Reply:
I would avoid alpine because there's a lot of bugs due to it using a different set of libraries for the linux. Slim should hopefully work but do some testing yourself to check it out